configure.ac

```diff
@@ -29,7 +29,11 @@ AS_IF([test "x$enable_bzip2" != "xno" ],
 	 AC_CHECK_LIB([bz2], [BZ2_bzBuffToBuffCompress], [], [AC_MSG_ERROR([the libbz2 library is missing])])],
   [AC_DEFINE(SWUPD_WITHOUT_BZIP2,1,[Do not use bzip2 compression])]
 )
-
+AC_ARG_ENABLE(
+  [signing],
+  [AS_HELP_STRING([--enable-signing], [Enable signature check (disabled by default)])],
+  [AC_DEFINE([SIGNING], [], [Enable signature check as default])]
+)
```

Comment:
Since swupd-client is only doing signature verification, I would prefer --enable-signature-verification instead.

The define could be SIGNATURES.